packages/nimble-components/src/anchor/index.ts

```diff
@@ -0,0 +1,53 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    Anchor as FoundationAnchor,
+    applyMixins,
+    DelegatesARIALink,
+    AnchorOptions
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import { template } from './template';
+import type { AnchorAppearance } from './types';
+
+export { FoundationAnchor };
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor': Anchor;
+    }
+}
+
+/**
+ * A nimble-styled anchor
+ */
+export class Anchor extends FoundationAnchor {
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: underline-visible
+     */
+    @attr({ attribute: 'underline-visible', mode: 'boolean' })
+    public underlineVisible = false;
+
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: appearance
+     */
+    @attr
+    public appearance: AnchorAppearance;
+}
+applyMixins(Anchor, DelegatesARIALink);
```

Comment:
The foundation Anchor class already applies this mixin.. why apply it agin?
https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/anchor/anchor.ts#L178

Reply:
I didn't see it following the class definition so I thought I had to apply it myself. Removed now.